### PR TITLE
Add sample: Front Door Standard/Premium (Preview)

### PR DESCRIPTION
**Pull request by johndowns**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Add sample: Front Door Standard/Premium (Preview)

### CODE REVIEW

**Comment by azure-quickstarts** [comment]
@johndowns - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

File: 201-front-door-standard-premium/main.bicep [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    priority: 1
+    weight: 1000
+  }
+}
+
+resource route 'Microsoft.Cdn/profiles/afdEndpoints/routes@2020-09-01' = {
+  name: routeName
+  parent: endpoint
+  dependsOn:[
+    origin // This explicit dependency is required to ensure that the origin group is not empty when the route is created.
```
Comment (bmoore-msft) [inline 201-front-door-standard-premium/main.bicep:71]: not sure I follow this one - how the route needs the the origin when there doesn't appear to be any relationship between them...

Comment (johndowns) [reply 201-front-door-standard-premium/main.bicep:71]: Yes, this is a known issue in the Front Door RP. You can't deploy a route until you have an origin group with an origin in it. I'm in discussions with the PG about trying to fix this :)

Comment (johndowns) [reply 201-front-door-standard-premium/main.bicep:71]: FYI this is the case on all of the samples. If the PG does fix it, I will update them all.

**Pull request merged**